docs/multicluster/user-guide.md

```diff
@@ -486,6 +487,20 @@ exported a Service: `default/nginx` with TCP Port `80`, other clusters can only
 export the same Service with the same Ports definition including Port names. At
 the moment, Antrea Multi-cluster supports only IPv4 multi-cluster Services.
 
+By default, a multi-cluster Service will use the exported Services' ClusterIPs (the
+original Service ClusterIPs in the export clusters) as the endpoints. Since
+Antrea v1.9.0, Antrea Multi-cluster also supports using the backend Pod IPs as
+the multi-cluster Service endpoints. You can change the value of configuration option
+`endpointIPType` in ConfigMap `antrea-mc-controller-config` from `ClusterIP`
+to `PodIP` to use Pod IPs as endpoints. All member clusters in a ClusterSet should
+use the same endpoint type. Existing ServiceExports should be re-exported after
+changing `endpointIPType`. `ClusterIP` type requires that Service CIDRs (ClusterIP
+ranges) must not overlap among member clusters, and always requires Multi-cluster
+Gateways to be configured. `PodIP` type requires Pod CIDRs must not overlap among
```

Comment:
```suggestion
Gateways to be configured. `PodIP` type requires Pod CIDRs not to overlap among
```

Reply:
The suggestion makes sense. Let me commit it.